### PR TITLE
add 2minsDevsNotes to awesome-tech-blogs

### PR DESCRIPTION
**Pull request by vtcaregorodtcev**

### CODE REVIEW

File: data.js [hunk 1]
```diff
@@ -1,4 +1,22 @@
 module.exports = [
+  {
+    name: "2minsDevsNotes",
+    description: "Github-based fast-to-read blog about dev's daily routine challenges",
+    url: "https://github.com/vtcaregorodtcev/2minsDevsNotes",
```
Comment (naresh-io) [inline data.js:5]: Please add your blog link here

Comment (naresh-io) [inline data.js:5]: Please add your blog link here

**Review by naresh-io — CHANGES_REQUESTED** [review]

File: data.js (hunk 1 above)
Comment (vtcaregorodtcev) [reply data.js:5]: this is it ;)

Comment (vtcaregorodtcev) [reply data.js:5]: this is github-based blog

Comment (vtcaregorodtcev) [reply data.js:5]: @naresh-io you can visit it and see :)

Comment (naresh-io) [reply data.js:5]: @markodenic can you please look into this?

Comment (vtcaregorodtcev) [reply data.js:5]: also, there is an article, about why writing articles on github is a very convenient way :) https://github.com/vtcaregorodtcev/2minsDevsNotes/issues/4

Comment (naresh-io) [reply data.js:5]: I appreciate the reference link, but I'm unsure if GitHub based blogs are currently allowed. @markodenic will review it.

Comment (vtcaregorodtcev) [reply data.js:5]: @naresh-io @markodenic any updates so far?

**Pull request merged**